MapboxNavigation/RouteMapViewController.swift

```diff
@@ -534,6 +534,8 @@ extension RouteMapViewController: NavigationComponent {
         }
         
         if routeLineTracksTraversal {
+            mapView.updateUpcomingRoutePointIndex(routeProgress: progress)
+            mapView.updateTraveledRouteLine(point: location.coordinate)
             mapView.updateRoute(progress)
```

Comment:
As discussed, lets consider moving `updateUpcomingRoutePointIndex(routeProgress:)` into `updateRoute(_:)`. It'll provide single point of entry for vanishing route line updates.

Reply:
Thanks for the suggestion! I'll remove the unuseful code.